### ISSUE TITLE
Predefined word vectors for RNN / LSTM

### ISSUE THREAD

**Issue by denmoroz** [fchollet/keras#1327]
In all of the examples Embedding layer is used. Is it possible to learn on variable length sequences of predefined word vectors, without learning them and without padding?

**Comment by dandxy89** [comment]
Take a look at:
*  [Padding Sequences](http://keras.io/preprocessing/sequence/#pad_sequences) - can be used for dealing with variable length sequences
* Word Vectors discussion/implementation - #853

**Comment by denmoroz** [comment]
Thanks! Word vectors discussion seems very useful for tasks, where modification of pretrained vectors is needed.

But It is not clear for me how to use padding for sequence of vectors? Suppose I don't want to use Embedding layer and just want to pass sequences of fixed word vectors to RNN. So, as I understand, I should pass X_train of shape (num_sentences, max_tokens_in_sentence, word2vec_dim). Also I must add padding for all sentences with tokens < max_tokens_in_sentence. Is it ok if I'll just use zero-vectors as padding?

I mean in Python:
```
X = np.zeros((len(sentences), max_sent_len, 300), dtype=np.float32)

for sent_idx, sent_vectors in enumerate(sentences):
    for time_idx, vector_array in enumerate(sent_vectors):
        X[sent_idx, time_idx, :] = vector_array
```

**Comment by liyi193328** [comment]
@denmoroz 
How is padding zero-vectors in your context?
In my situation, loss value is always nan
can you help me?
#1360
